.env.example

```diff
@@ -10,11 +10,15 @@ ETHEREUM_NODE_WS=ws://localhost:8545
 # Service Endpoints
 API_SERVICE_EXTERNAL=http://api.circles.local
 GRAPH_NODE_EXTERNAL=http://graph.circles.local
+PATHFINDER_SERVICE_ENDPOINT=http://localhost:8081
 RELAY_SERVICE_EXTERNAL=http://relay.circles.local
 
 # Database Endpoints
 DATABASE_SOURCE=graph
 
+# Pathfinder Type 'cli' or 'server'
```

Comment:
Could you describe here what it is used for? Could we also update internal docs (there is a section with env environments) ?

Reply:
great, there is a typo - 'steps' instead of 'stpes' I approve